src/components/search/search.tsx

```diff
@@ -141,31 +268,40 @@ const Search: React.FunctionComponent<SearchProps> = ({
                 </div>
             )}
             <div sx={{}}></div>
-            {!!numberOfResults || isValidating || error ? (
+            {!!resultData.length || resultIsValidating || error ? (
                 <>
                     <Results
-                        data={data}
-                        isLoading={isValidating}
+                        data={resultData}
+                        isLoading={isLoading}
                         hasError={error}
                         layout={resultsLayout}
                     />
-                    {!fullyLoaded && (
+                    {showLoadMoreButton && (
                         <div
                             sx={{
                                 display: 'flex',
                                 justifyContent: 'center',
                                 marginTop: ['inc70', null, 'inc90'],
                             }}
                         >
-                            {!isValidating && data && (
-                                <Button
-                                    onClick={() =>
-                                        setResultsToShow(resultsToShow + 10)
+                            {!resultIsValidating && resultData && (
+                                <a
+                                    href={
+                                        hasEmptyFilterAndQuery(
+                                            searchString,
+                                            allFilters
+                                        )
+                                            ? `/developer${path}/?page=${
+                                                  currentPage + 1
+                                              }`
+                                            : '#'
                                     }
```

Comment:
nitpick, but might be better to calculate this value above the render method to keep the DOM structure easy to read.